Casks/openzfs.rb

```diff
@@ -4,8 +4,17 @@
 
   url "https://openzfsonosx.org/w/images/#{version.after_comma[0]}/#{version.after_comma}/OpenZFS_on_OS_X_#{version.before_comma}.dmg"
   name "OpenZFS on OS X"
+  desc "Open source port of OpenZFS"
   homepage "https://openzfsonosx.org/"
 
+  livecheck do
+    url "https://openzfsonosx.org/wiki/Downloads"
+    strategy :page_match do |page|
+      match = page.match(%r{href=['"]?/w/images/\d/(\h+)/openzfs[_-]on[_-]os[_-]x[._-](\d+(?:\.\d+)*)\.dmg}i)
+      "#{match[2]},#{match[1]}"
```

Comment:
```suggestion
      page.scan(%r{href=['"]?/w/images/\d/(\h+)/openzfs[_-]on[_-]os[_-]x[._-](\d+(?:\.\d+)*)\.dmg}i).map do |match|
        "#{match[1]},#{match[0]}"
      end
```

As above, this suggestion uses `#scan` instead of `#match`.